### PR TITLE
update skupper readme

### PR DESCRIPTION
**Pull request by pwright**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -14,31 +14,31 @@
...
 == Building and testing
 
-NOTE: Dispatch will not build on Windows.
+NOTE: You cannot build Skupper router on Windows.
 
 === Dependencies
 
-To build dispatch on a yum-based Linux system, you will need the following
-packages installed:
+To build dispatch on a yum-based Linux system, you need the following packages installed:
```
Comment (kgiusti) [inline README.adoc:41]: dispatch

File: README.adoc [hunk 2]
```diff
@@ -154,50 +152,53 @@ cmake .. -DPYTHON_TEST_COMMAND='-m;pytest;-vs;--junit-xml=junitxmls/${py_test_mo
...
 
 === Run Time Validation
 
 The CTest test suite can be configured to enable extra run time
 validation checks against the dispatch router.
 
-Since run time validation slows down qdrouter considerably it is
-disabled by default.  It can be enabled by setting the RUNTIME_CHECK
-build flag via the cmake command.
+Since run time validation slows down `skrouter` considerably it is disabled by default.
```
Comment (kgiusti) [inline README.adoc:196]: skrouter should be skrouterd actually
Also 'dispatch' used on line # 194

File: README.adoc [hunk 3]
```diff
@@ -313,7 +304,7 @@ Other options include `Debug` (disables optimizations) and `Coverage`.
 
 |`-DCMAKE_INTERPROCEDURAL_OPTIMIZATION=ON`
 |With CMake 3.9+, compiles the project with LTO (Link Time Optimization) enabled.
-Older CMake will only honor this option with the Intel compiler on Linux.
+Older versions of `CMake`` only honor this option with the Intel compiler on Linux.
```
Comment (kgiusti) [inline README.adoc:307]: Not this line, but 'Dispatch' is used on line 296 
Stoopid github not letting me comment on unchanged lines... :(

Comment (kgiusti) [reply README.adoc:307]: Oh, and line 310 as well....

**Review by kgiusti — CHANGES_REQUESTED** [review]
A couple of dispatch-related names slipped through - see inline.

File: README.adoc [hunk 4]
```diff
@@ -14,31 +14,31 @@
...
+It provides flexible and scalable interconnect between any AMQP endpoints, and is forked from the https://github.com/apache/qpid-dispatch/[qpid-dispatch] project to focus on the https://skupper.io[Skupper] use case.
 
-Qpid Dispatch is a high-performance, lightweight AMQP 1.0 message router.
-It provides flexible and scalable interconnect between any AMQP endpoints,
-whether they be clients, brokers or other AMQP-enabled services.
 
 == Building and testing
 
-NOTE: Dispatch will not build on Windows.
+NOTE: You cannot build Skupper router on Windows.
```
Comment (jiridanek) [inline README.adoc:37]:
```suggestion
NOTE: Skupper router only supports the Linux OS.
```

This is a change in content, but I think that's the way we should describe this. Skupper is Linux-only and it is not ashamed of it.

File: README.adoc [hunk 5]
```diff
@@ -111,24 +109,23 @@ The TCP system tests run only if Python selectors is available.
 pip3 install --user quart h2 selectors
 ----
 
-Websocket system tests use the Python websockets asyncio module.
+Websocket system tests use the Python `websockets asyncio` module.
```
Comment (jiridanek) [inline README.adoc:112]: Name of the module is `websockets`, https://pypi.org/project/websockets/. The module does use asyncio API but that is not part of it's name.

File: README.adoc (hunk 4 above)
Comment (pwright) [reply README.adoc]: I think it's a good change, and have implemented it

File: README.adoc [hunk 6]
```diff
@@ -276,21 +270,18 @@ To enable the address sanitizer set the RUNTIME_CHECK build flag to "asan":
...
 ----
 cmake .. -DCMAKE_C_FLAGS=-DQD_MEMORY_DEBUG -DRUNTIME_CHECK=hwasan
 ----
 
-The ASAN (libasan) and UBSAN (libubsan) libraries must be installed in
-order to use this option.
-
-NOTE: Memory pool will produce false leak reports unless `QD_MEMORY_DEBUG`
-is also defined.
+NOTE: The memory pool produces false leak reports unless `QD_MEMORY_DEBUG` is also defined.
```
Comment (jiridanek) [inline README.adoc:282]: `QD_MEMORY_DEBUG` now defines itself, @kgiusti we could safely delete this line, if we wanted. Maybe best to leave it around; people will not read this anyways (as witnessed by the number of people (me included) who struggled when this was not defined automatically).

Comment (jiridanek) [reply README.adoc:282]: lets not touch this, it is not important for the topic of the PR

**Review by kgiusti — APPROVED** [review]
Ship it!

**Pull request merged**